src/Illuminate/Foundation/Application.php

```diff
@@ -1051,7 +1051,95 @@ public function setLocale($locale)
      */
     public function isLocale($locale)
     {
-        return $this->getLocale() == $locale;
+        return $this->getLocale() === $locale;
+    }
+
+    /**
+     * Get default application locale
+     *
+     * @return string
+     */
+    public function getDefaultLocale()
+    {
+        return config('app.fallback_locale');
+    }
+
+    /**
+     * Get additional locales supported by application
+     *
+     * @return array
+     */
+    public function getAdditionalLocales()
+    {
+        return config('app.additional_locales');
```

Comment:
There could be a confusion because locales could also mean all app locales including app.locale. I found additional_locales or supported_locales tells a bit more but realy don't have any opinion on that.